azure-pipelines/run_test_steps.yml

```diff
@@ -65,7 +65,7 @@ steps:
     displayName: 'Run tests'
     continueOnError: ${{ parameters.allowFailure }}
     env:
-      TWISTED_REACTOR: ${{ parameters.windowsReactor }}
+      TRIAL_ARGS: --reactor=${{ parameters.windowsReactor }}
```

Comment:
I guess that once we are happy with GHA, we can remote the Windows tests from Azure

Reply:
OK. I will remove the Windows CI from Azure.

And I will also need to update the branch protection via GitHub UI not to require Azure Windows.